packages/puppeteer-extra-plugin-adblocker/src/index.ts

```diff
@@ -99,14 +102,23 @@ export class PuppeteerExtraPluginAdblocker extends PuppeteerExtraPlugin {
     if (this.blocker === undefined) {
       try {
         this.blocker = await this.loadFromCache()
+        this.setRequestInterceptionPriority()
       } catch (ex) {
         this.blocker = await this.loadFromRemote()
+        this.setRequestInterceptionPriority()
         await this.persistToCache(this.blocker)
       }
     }
     return this.blocker
   }
 
+  /**
+   * Sets the request interception priority on the `PuppeteerBlocker` instance.
+   */
+  private setRequestInterceptionPriority(): void {
+    this.blocker?.setRequestInterceptionPriority(this.opts.interceptResolutionPriority)
```

Comment:
Not familiar enough with the lib to know if this is correct, it seems to be passing it down a layer?

Reply:
@benallfree Yes exactly, it references the instance of `adblocker-puppeteer` that is used internally, which is the one that really intercepts and handles the requests, and this `pupeteer-extra` plugin acts as a wrapper: https://github.com/ghostery/adblocker/blob/master/packages/adblocker-puppeteer/adblocker.ts#L282-L283